### ISSUE TITLE
Options aren't shown and can't be selected if `format_item` is not set

### ISSUE THREAD

**Issue by Diegovsky** [nvim-telescope/telescope-ui-select.nvim#10]
I don't know if you can reproduce this, but on my setup, doing `:lua vim.ui.select({0, 2, 3}, {prompt="Choose"}, print)` yields a picker with no options and the werdest part is that it knows the list length:
![image](https://user-images.githubusercontent.com/46163903/164982027-d55e8edc-9f8e-4284-b36b-2881efcef5bb.png)

However, if I instead set the `format_item` field to `tostring`, it works:
![image](https://user-images.githubusercontent.com/46163903/164982092-88f0adbc-aca7-4a46-ae1c-96c95c093868.png)

To quote `:help vi.ui.select`:
```
                                 • format_item (function item -> text)
                                   Function to format an individual item from
                                   `items`. Defaults to `tostring`.
```

**Comment by Conni2461** [comment]
my bad fixed in #7 

thanks for reporting

**Comment by Diegovsky** [comment]
Great! Thanks for the very fast fix :+1:
